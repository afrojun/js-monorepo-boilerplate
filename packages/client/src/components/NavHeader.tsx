import React from 'react'

const NavHeader = () => (
  <div className="flex flex-row justify-between px-1 py-1 bg-slate-700 text-white">
    JS Monorepo Boilerplate
  </div>
)

export default NavHeader
