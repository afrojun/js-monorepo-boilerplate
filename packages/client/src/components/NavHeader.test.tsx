import React from 'react'
import { render, screen } from '@testing-library/react'
import NavHeader from './NavHeader'

describe('NavHeader', () => {
  it('renders the logo', () => {
    render(<NavHeader />)
    expect(screen.getByText('JS Monorepo Boilerplate')).toBeInTheDocument()
  })

  it('matches the snapshot', () => {
    expect(render(<NavHeader />).asFragment()).toMatchSnapshot()
  })
})
