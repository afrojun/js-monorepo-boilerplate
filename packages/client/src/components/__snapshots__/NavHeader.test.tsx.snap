// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NavHeader matches the snapshot 1`] = `
<DocumentFragment>
  <div
    class="flex flex-row justify-between px-1 py-1 bg-slate-700 text-white"
  >
    JS Monorepo Boilerplate
  </div>
</DocumentFragment>
`;
